internal/ssh/internal/mock/log.go

```diff
@@ -0,0 +1,21 @@
+package mock
+
+var log Logger = noopLogger{}
```

Comment:
What uses this logger? Why do we want to ignore its logs?

Reply:
We already get BYO logger support from slog and its pluggable handler right? Is there a reason not to always "log" and leave it up to the handler whether it gets handled or not?